scripts/init.js

```diff
@@ -37,6 +37,13 @@ module.exports = function(hostPath, appName, verbose) {
   // Copy the files for the user
   fs.copySync(path.join(selfPath, 'template'), hostPath);
 
+  // Rename files
+  [
+    ['gitignore', '.gitignore'],
```

Comment:
Please don't over abstract things. There's only one case right now so you can just write

```
fs.move(path.join(hostPath, 'gitignore'), path.join(hostPath, '.gitignore'));
```

which is going to be clearer